Trakt/Api/Enums/TraktItemType.cs

```diff
@@ -0,0 +1,26 @@
+#pragma warning disable SA1300
+#pragma warning disable CA1707
```

Comment:
since these are new files it might be good to solve the warnings with decorators instead of disabling them

Reply:
We can't really do that because `System.Text.Json` currently does not support manually setting the json representation of an enum value.